js/components/dashboard/dashboard.js

```diff
@@ -120,21 +121,29 @@ export default class Dashboard extends PureComponent {
     const {
       activities, students, studentProgress, expandedStudents,
       expandedActivities, setActivityExpanded, setStudentExpanded,
-      setStudentsExpanded, setQuestionExpanded, expandedQuestions
+      setStudentsExpanded, setQuestionExpanded, expandedQuestions,
+      selectedQuestion, selectQuestion
     } = this.props
     const anyStudentExpanded = expandedStudents.includes(true)
     const anyQuestionExpanded = expandedQuestions.includes(true)
     const showTallQuestionHeader = anyStudentExpanded || anyQuestionExpanded
     const activitiesList = activities.toList().filter(activity => activity.get('visible'))
     return (
       <div className={css.dashboard}>
+        <QuestionDetails
+          selectedQuestion={selectedQuestion}
+          onClose={(e) => selectQuestion(Map())}
```

Comment:
This is my typical React comment - I try to avoid arrow expressions in the render method. A new function is created each time render() is called, so QuestionDetails will be re-rendered each time too, even if its props are not changed. Most likely not causing any issues here, but personally I'd avoid this in general - to keep consistent style and one day it can cause real issues.